views/layouts/basic.blade.php

```diff
@@ -1,9 +1,8 @@
-{{-- TODO: Change below to @php when Laravel is upgraded --}}
-<?php
+@php
   $primaryColor = $settings->get('theme_primary_color', '#000');
-?>
+@endphp
```

Comment:
from local testing it seems like declaring variables within `@php` doesn't actually work for some reason 🤔

Reply:
oh, it works without the comment on top nvm